src/test/java/logintests/PositiveLoginTest.java

```diff
@@ -14,33 +14,39 @@
 
     @Test
     public void logInTestUsingSignInButton() {
+        log.info("Starting Log In test using Sign in button");
+
         BasePage basePage = new BasePage(driver);
         openURL(ConstantsURLs.baseURL);
         basePage.clickOnSignInLink();
 
         LoginPage loginPage = new LoginPage(driver);
         MyAccountPage myAccountPage = loginPage.logInUserSignInBtn(ConstantsCredentials.validEmail,
                 ConstantsCredentials.validPassword);
+        log.info("User successfully logged in");
 
-        Assert.assertEquals(ConstantsURLs.myAccountURL, myAccountPage.getCurrentUrl());
-        Assert.assertEquals(ConstantsMessages.myAccountTitle, myAccountPage.getCurrentPageTitle());
+        Assert.assertEquals(myAccountPage.getCurrentUrl(), ConstantsURLs.myAccountURL);
+        Assert.assertEquals(myAccountPage.getCurrentPageTitle(), ConstantsMessages.myAccountTitle);
         Assert.assertTrue(basePage.isSignOutLinkVisible());
-        Assert.assertEquals(ConstantsMessages.validUsername, basePage.getUsername());
+        Assert.assertEquals(basePage.getUsername(), ConstantsMessages.validUsername);
```

Comment:
just as a suggestion for future tests, you can add log messages also for assertions